src/client/sandbox/node/index.js

```diff
@@ -101,4 +107,36 @@ export default class NodeSandbox extends SandboxBase {
         this.win.attach(window);
         this.element.attach(window);
     }
+
+    static processSelector (selector) {
+        var attributeSelectorRegEx = /\[([\w-^]+)=(.+?)]/g;
+        var stringifySelector      = function (value) {
+            if (value === null)
+                return 'null';
+
+            else if (value === void 0)
+                return 'undefined';
+
+            return selector;
+        };
+
+        selector = stringifySelector(selector);
+
+        return selector.replace(attributeSelectorRegEx, (str, name, value) => {
```

Comment:
In case then we don't have attribute selector is `replace` equivalent in speed  to `test`? Can you measure using https://jsperf.com/? If not, we should add basic filter with `test`